management/azure_mgmt_compute/azure_mgmt_compute.gemspec

```diff
@@ -5,12 +5,12 @@
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
-require 'generated/azure_mgmt_compute/module_definition'
-require 'generated/azure_mgmt_compute/version'
+require '2017-03-30/generated/azure_mgmt_compute/module_definition'
```

Comment:
This is pretty cool. How did you do that is selects the latest version here and in the version as well :)

Reply:
Hehe, the gemspec file is not autogenerated at the moment, as it wasn't before, so I updated it. Let's evaluate whether we want to to this automatically and where, because we don't generate this via autorest today, maybe we can do it with the "uber" client generation.